python/setup.cfg

```diff
@@ -28,6 +28,3 @@ per-file-ignores = tests/*:S101
 
 [aliases]
 test = pytest
-
-[tool:pytest]
-collect_ignore = ['setup.py']
```

Comment:
why remove this?

Reply:
Test tools give warnings that this setting is present, but is not necessary

Reply:
```
========================================================== warnings summary ===========================================================
../../../.pyenv/versions/3.9.6/envs/nessie@3.9.6-dev/lib/python3.9/site-packages/_pytest/config/__init__.py:1233
  /home/dmitri/.pyenv/versions/3.9.6/envs/nessie@3.9.6-dev/lib/python3.9/site-packages/_pytest/config/__init__.py:1233: PytestConfigWarning: Unknown config option: collect_ignore
  
    self._warn_or_fail_if_strict(f"Unknown config option: {key}\n")

-- Docs: https://docs.pytest.org/en/stable/warnings.html
```

Reply:
interesting, must have been deprecated in a version bump :+1: